catroid/src/main/java/org/catrobat/catroid/ui/fragment/ScriptFragment.java

```diff
@@ -498,6 +517,9 @@ public void onClick(DialogInterface dialog, int which) {
 	}
 
 	private void handleContextMenuItemClick(int itemId, BrickBaseType brick, int position) {
+		if (listView.isCurrentlyHighlighted()) {
```

Comment:
this should not be necessary right? because once you open the context menu the highlighting should have been removed already. I understand that it sometimes makes sense to be careful and add extra checks for certain states that you think should not be possible. however, I think in this case you should be sure if this can happen or not.